### PR TITLE
feat: Add spec sys_cpuset_cpus

### PR DESCRIPTION
**Pull request by wushiqinlou**
Signed-off-by: jiazhang <jiazhang@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This spec is used to add container check on one insights-plugins advisor rule

### CODE REVIEW

**Comment by xiangce** [comment]
This PR seems to be duplicated with the #3612

**Comment by wushiqinlou** [comment]
@xiangce - Please reivew.

**Review by xiangce — APPROVED** [review]
Good to me

File: insights/tests/parsers/test_cpuset_cpus.py [hunk 1]
```diff
@@ -1,13 +1,43 @@
 from insights.parsers import cpuset_cpus
 from insights.tests import context_wrap
-
+import doctest
```
Comment (ryan-blakley) [inline insights/tests/parsers/test_cpuset_cpus.py:3]: Could you please move this above the absolute imports.

```
import doctest

from insights.parsers import cpuset_cpus
from insights.tests import context_wrap
```

File: insights/parsers/cpuset_cpus.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+========================================================
 
 This parser reads the content of ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
 This file shows the default cgroup cpuset.cpu of system. The format
 of the content is string including comma.
 """
 
-from .. import parser, CommandParser
+from insights import ContainerParser
+from insights import parser, CommandParser
```
Comment (ryan-blakley) [inline insights/parsers/cpuset_cpus.py:11]: Could you change these to full path imports please?

```
from insights.core import CommandParser, ContainerParser
from insights.core.plugins import parser
```

**Review by ryan-blakley — CHANGES_REQUESTED** [review]

File: insights/tests/parsers/test_cpuset_cpus.py (hunk 1 above)
Comment (wushiqinlou) [reply insights/tests/parsers/test_cpuset_cpus.py]: @ryan-blakley - Updated, please check

File: insights/parsers/cpuset_cpus.py (hunk 2 above)
Comment (wushiqinlou) [reply insights/parsers/cpuset_cpus.py]: @ryan-blakley - Updated, please check

File: insights/tests/parsers/test_cpuset_cpus.py (hunk 1 above)
Comment (ryan-blakley) [reply insights/tests/parsers/test_cpuset_cpus.py]: Thank you.

File: insights/parsers/cpuset_cpus.py (hunk 2 above)
Comment (ryan-blakley) [reply insights/parsers/cpuset_cpus.py]: Thank you.

**Review by ryan-blakley — APPROVED** [review]
Everything looks good to me.

**Pull request merged**